src/pcrlock/pcrlock.c

```diff
@@ -3788,14 +3790,17 @@ static int event_log_reduce_to_safe_pcrs(EventLog *el, uint32_t *pcrs) {
                 continue;
 
         drop:
+                if (arg_strict && FLAGS_SET(*pcrs, UINT32_C(1) << pcr))
+                        dropped_relevant_pcr |= pcr;
+
                 *pcrs &= ~(UINT32_C(1) << pcr);
 
                 if (strextendf_with_separator(&dropped, ", ", "%" PRIu32 " (%s)", pcr, tpm2_pcr_index_to_string(pcr)) < 0)
                         return log_oom();
         }
 
         if (dropped)
-                log_notice("PCRs dropped from protection mask: %s", dropped);
+                log_full(dropped_relevant_pcr ? LOG_ERR : LOG_NOTICE, "PCRs dropped from protection mask: %s", dropped);
```

Comment:
we have this coding style rule that we only use C's downgrade-to-bool feature for stuff that is boolean or boolean-like, as well as pointers. for arithmetic values/bitmasks and such do explicit comparisons however, i.e. `dropped_relevant_pcr != 0`